src/main/java/uk/gov/justice/digital/delius/utils/ContentDispositionHeader.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.delius.utils;
+
+import lombok.Getter;
+
+import java.nio.charset.StandardCharsets;
+
+import static java.net.URLEncoder.encode;
+
+
+@Getter
+public class ContentDispositionHeader {
```

Comment:
Did you look into using the [org.springframework.http.ContentDisposition](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/ContentDisposition.Builder.html) class for this?  It provides a builder pattern and seems to do most of what you're doing here out of the box.